lightning-tag/implementation-validations.md

```diff
@@ -0,0 +1,52 @@
+# Post-Implementation Validation
+
+After Lightning Tag has been implemented, how do I confirm it was implemented correctly?
+
+## Lightning Tag Validations
+
+Open your browsers DevTools and load/refresh the page that you want to test.
+
+### Validate 1st Impression Call
+
+In DevTools, Switch to the Network tab and type `crwdcntrl`  in the filter field to show calls on your page to the Lotame Lightning Tag. Look for `bcp.crwdcntrl.net/6/data`
+
+![Network Tab filtered for Lightning Tag calls](../images/lt-devtools-validation-bcp1.png)
+
+Select that row and then look at the response tab. The Response should look something like:
+
+`{"pid":"7de1b12b97e8ef15f61ccfcbe59368ca","tc":["target_code1","target_code2","all"],"spx":[61,14,81,8,5,115]}`
+
+Whereas the `pid` is the Lotame profile ID and the `tc` containsTargeting Codes (or Audience IDs)
```

Comment:
for audience ids it would be in aud param, not tc

Reply:
Noah, I'm confused then, my screenshot shows `tc`. What is the issue here? That I should remove the audience IDs part in parentheses?

Reply:
To me it sounds like either audienceIds or targeting codes would appear in the tc param. I think it would be clearer to put in the () the following: (or Audience Ids in the `aud` parameter)